flink-streaming-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSink.java

```diff
@@ -35,24 +36,28 @@
 	private static final Logger LOG = LoggerFactory.getLogger(RMQSink.class);
 
 	private String QUEUE_NAME;
```

Comment:
I know its not part of the issue's scope, but could you rename this field to `queueName`. Otherwise, the class has a very poor style which makes it hard to read it.